typings/declarations.d.ts

```diff
@@ -1,36 +1,71 @@
 declare module "yoffee" {
-
     /**
      * Returns a `DocumentFragment` that has a one-way binding to each of the objects.
-     * @param {...Object} propsObjects Holds the state of this yoffee element
-     * @return a document fragment
+     * @param {object} propsObjects Objects which are used in the expressions inside the html for updating the html.
+     * @returns {(strings: string[], ...expressionCbs: Function[]) => DocumentFragment} html bound to `propsObjects` as a DocumentFragment.
      */
-    function html(...propsObjects: object[]): any; 
+    function html(...propsObjects: object[]): (strings: TemplateStringsArray, ...expressionCbs: Function[]) => DocumentFragment;
+
+    interface YoffeeElementWrapper extends YoffeeElement {
```

Comment:
thanks for complementing the declaration, there were parts I wasn't sure about.

Reply:
But those functions may not be exposed publicly, I thought they should not, although of course developers may access them.
I kept the declaration minimal also because I wasn't sure what was intentionally exposed/exported vs what was overlooked.

Reply:
You're talking about `html` or the `YoffeeElementWrapper`? They're both exposed publicly.
`html` needs to show an error when the expression is not a function so that's crucial.
`YoffeeElementWrapper` helps the user find `onPropUpdate`, `onConnect` and `onDisconnect` callbacks

Reply:
talking about `YoffeeElementWrapper.`. But I concede I had not fully understood what it was about :)